build-xcframework.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env sh
 
 NAME="anoncreds"
-VERSION="0.1.0-dev.4"
+VERSION="0.1.0-dev.5"
```

Comment:
I will do the same fix here as I did for askar. We will not have to update this ourselves anymore.